tools/api-markdown-documenter/prettier.config.cjs

```diff
@@ -1,6 +1,10 @@
 module.exports = {
     ...require("@fluidframework/build-common/prettier.config.cjs"),
-    importOrder: ["<THIRD_PARTY_MODULES>", "^[./]"],
+    importOrder: [
+        "^node:(.*)$", // Special-case `node:` imports
```

Comment:
This ensures "node:foo` imports are sorted where eslint wants them. This is also consistent with my future plans to use this plugin more widely. See #11988